README.md

```diff
@@ -41,9 +41,11 @@ Snapshots is a WP CLI package. An environment with WP CLI enabled is required, a
 The recommended method for installing the package is directly from the GitHub repository:
 
 ```bash
-wp package install 10up/snapshots
+wp package install 10up/snapshots:@stable
```

Comment:
Just noting that I was only able to install the package using the `:@stable` suffix